Self-host Weekly IRC meeting ical file and change meeting location from freenode -> libera.chat (#1116)
bitcoin-core/bitcoincore.org: _posts/en/pages/2016-01-01-meetings.md

- General developer meeting: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/General-IRC-meeting

> [Here as ical format][meeting calendar ical] to subscribe via a calendar app.

Reviewer: nit: could use a touch-up
```suggestion
You can use the [`iCal`][meeting calendar ical] file to subscribe via your calendar app.
```
